src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -60,6 +62,14 @@ class DeployTask extends AbstractServerTask {
         })
     }
 
+    private Boolean container = null;
+
+    @Option(option = 'container', description = 'Run the server in a Docker container instead of locally. The default value is false.')
+    void setContainer(boolean container) {
```

Comment:
Is this container option available for someone not using the `devc` goal to set and deploy to a container? Would that work?

Reply:
It would cause the loose war config file to be generated using the project variable io.openliberty.tools.projectRoot which could be put to some productive use but it is not really meant to be used this way. In particular you would have to define the variable (in bootstrap.properties, server.xml or command line) manually to run the app in the server. 
Another way to pass the information would be a Java System property. Or a temp file. What makes the most sense in a Gradle environment? Are there other situations that require an option value in multiple tasks?

Reply:
@mattbsox Can you chime in here? I don't thinking having this container option exposed on the DeployTask makes sense.

Reply:
I think passing the value as a project property on the command line would work.

Should be able to set it like this: `-Pliberty.dev.container=true`

All the tasks have access to the Liberty extension so it will be shared across them. You can use the same line in the `runGradleTask()` parameters as well.

Reply:
This has been changed as suggested above to use a project property and access the Liberty extension.